samples/core/guide/autograph.ipynb

```diff
@@ -123,32 +165,54 @@
     },
     {
       "metadata": {
-        "id": "ohbSnA79mcJV",
+        "id": "Hh1PajmUJMNp",
         "colab_type": "text"
       },
       "cell_type": "markdown",
       "source": [
-        "## Automatically converting control flow\n",
-        "\n",
-        "AutoGraph can convert a large chunk of the Python language into equivalent graph-construction code, and we're adding new supported language features all the time. In this section, we'll give you a taste of some of the functionality in AutoGraph.\n",
-        "AutoGraph will automatically convert most Python control flow statements into their correct graph equivalent.  "
+        "We'll enable  [eager execution](https://www.tensorflow.org/guide/eager) for demonstration purposes, but AutoGraph works in both eager and [graph execution](https://www.tensorflow.org/guide/graphs) environments:"
```

Comment:
> works in both eager and graph

I think we need to be careful here not to give the impression that it does a graph-run automatically in eager.

It transforms the code to something that could be used in tin graph construction. It will run correctly in eager, but it is not implicitly applying a `tfe.Defun`.